src/lib/types.ts

```diff
@@ -88,6 +89,7 @@ export interface SendBirdStateConfig {
   isReactionEnabled: boolean;
   isMentionEnabled: boolean;
   isVoiceMessageEnabled?: boolean;
+  uikitMultipleFilesMessageLimit: Nullable<number>; // shouldn't be nullable
```

Comment:
We have to set a default value, isn't it? @liamcho

Reply:
ah yes you are right. Thanks for the catch. Let me create fix pr for this.
<img width="399" alt="Screen Shot 2023-09-11 at 2 55 13 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/16806397/ae99014b-173d-4ffe-a787-ca7baf8e81b6">

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/728